de/tests/test_factorization.py

```diff
@@ -81,5 +81,8 @@ def test_split_data():
     data = ImportMelanoma()
     train_X, test_X = split_data(data)
     full_X = impute(train_X)
+    full_X_lin = impute(train_X, linear=True)
 
-    print(ma.corrcoef(ma.masked_invalid(full_X.flatten()), ma.masked_invalid(test_X.flatten())))
+    non_lin = ma.corrcoef(ma.masked_invalid(full_X.flatten()), ma.masked_invalid(test_X.flatten()))[0][1]
```

Comment:
Tests should check cases we can be absolutely sure will always be true. I'm not sure about that here.